dotnet/src/SemanticKernel.UnitTests/Utilities/IEnumerableExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel;
+
+public static class EnumerableExtensions
+{
+#pragma warning disable IDE1006 // Naming rule violation: Missing suffix: 'Async'
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+    public static async IAsyncEnumerable<T> ConvertToAsyncEnumerable<T>(this IEnumerable<T> source)
+    {
+        foreach (var item in source)
+        {
+            yield return item;
+        }
+    }
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+#pragma warning restore IDE1006 // Naming rule violation: Missing suffix: 'Async'
+}
```

Comment:
This already exists here:
https://github.com/microsoft/semantic-kernel/blob/1f6018dd21a2e64749a803a97d5184318ab21101/dotnet/src/InternalUtilities/src/Linq/AsyncEnumerable.cs#L33-L43

Reply:
The ToAsyncEnumerable in the unittest project is clashing with the referenced ones. I'm going to change the internal utilities to avoid that.